cli.js

```diff
@@ -23,6 +23,8 @@ var argv = yargs
   .usage('Usage: $0 add <username> <contribution>')
   .command('init', 'Prepare the project to be used with this tool')
   .usage('Usage: $0 init')
+  .command('check', 'Compares contributors from Github with the ones credited in .all-contributorsrc')
+  .usage('Usage: $0 check <repository>')
```

Comment:
Don't forget to remove `<repository>` when you refactor this to use `projectOwner` and `projectName`